week-2/implement/contains.js

```diff
@@ -1,3 +1,7 @@
-function contains() {}
+function contains(item,prop) {
+  
+return (prop in item);
```

Comment:
Good work, but can you change the parameter item to more meaningful name so that it looks like it is a object. Because Item does not give meaning for an abject